metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -159,9 +159,9 @@ def test_flows(pytestconfig, flow_file_path: str) -> None:
     # `check_valid_logs_process` process.
 
     test_cmd = (
-        f"{_python()} {full_path} --datastore=s3 kfp run "
+        f"{_python()} {full_path} --datastore=s3 --with retry kfp run "
         f"--wait-for-completion --workflow-timeout 1800 "
-        f"--max-parallelism 3 --experiment metaflow_test --tag test_t1 "
+        f"--max-parallelism 5 --experiment metaflow_test --tag test_t1 "
```

Comment:
Seems like there are competing priorities in this PR, to me increasing the parallelism is likely to cause more issues. Happy to try to this but obviously this will just run things faster (as more is running in parallel) so in theory the peak load has the potential to be higher.

Reply:
Got it, but given the resource constraints we place of every pod, I think we will still remain below the threshold. The main reason I made this change was because sometime the tests are taking more than an hour (and thus timeout) because we have over 10 test flows now.

Reply:
Ah interesting! Didn't realize that, that's pretty crazy, do we know what the bottlenecks are because ideally the integration tests really shouldn't be doing much actual work so is it just in provisioning or waiting for retries?

Reply:
The bottleneck to my knowledge is that the integration tests spin off quite a few flows, so it takes K8s (especially during the daytime when many people are experimenting) to schedule the pods for the flow. Once the flow starts, it takes less than 5 minutes to complete, so its that initial pending state that's affecting everything.